### PR TITLE
up-194-users-have-profiles

### PR DESCRIPTION
**Pull request by SeanRichhh**
Added user profiles to the registration page/header/profile page. styled the profile page and resized all the images to fit.

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -27,6 +28,7 @@ const Header = ({ companyName }) => {
     navigate('../', { replace: true });
   };
 
+  console.log('heellooo' , loggedInUser);
```
Comment (Auenc) [inline src/components/Header/Header.jsx:31]: let's remove this console.log

File: src/components/posts/Posts.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ export default function Posts({
 	error,
 }) {
 	let commentLength = 0;
+	console.log("helloooo", posts);
```
Comment (Auenc) [inline src/components/posts/Posts.js:12]: let's remove this

**Review by Auenc — CHANGES_REQUESTED** [review]
Looks good! Just remove the redundant console logs and we'll be good to go!

**Review by Auenc — APPROVED** [review]
LGTM

**Review by sigristarisa — APPROVED** [review]

**Pull request merged**